hrms/www/hrms.py

```diff
@@ -21,5 +21,44 @@ def get_context_for_dev():
 
 def get_boot():
 	return frappe._dict(
-		{"site_name": frappe.local.site, "push_relay_server_url": frappe.conf.get("push_relay_server_url")}
+		{"site_name": frappe.local.site, "push_relay_server_url": frappe.conf.get("push_relay_server_url") or ""}
 	)
+
+
+@frappe.whitelist(allow_guest=True)
+def oauth_providers():
```

Comment:
Any reason for this function to be placed here instead of `hrms/api/__init__.py`?

Reply:
Also, can you add screenshots/video? 🙊